tests/openmetrics/test_parser.py

```diff
@@ -289,10 +289,9 @@ def test_no_metadata(self):
         self.assertEqual([metric_family], list(families))
 
     def test_empty_metadata(self):
-        families = text_string_to_metric_families("""# HELP a 
-# UNIT a 
-# EOF
-""")
+        families = text_string_to_metric_families("# HELP a \n"
+                                                  "# UNIT a \n"
```

Comment:
Seems like the parser requires the blank space here, which the end-of-line fixer I added in the `pre-commit` config wanted to remove from the multiline string. Possibly a separate bug in the parser? I don't know enough about the openmetrics format to say.

Reply:
The parser is correct, also changing things like this makes it harder to manually run these tests via copy&page. They're written this way for a reason.

Reply:
Good to know - lemme revert this part and remove the end-of-line fixer as a quick fix. There are ways to ignore particular files, but I don't love doing that, so it seems like that particular formatter needs some more thought.